Shared/Samples/Manage features/ManageFeaturesView.swift

```diff
@@ -0,0 +1,387 @@
+            data = .failure(error)
+        }
+    }
+    
+    @ViewBuilder
+    func mapView(_ data: Data) -> some View {
+        MapViewReader { mapView in
+            MapView(map: data.map)
+                .onAttributionBarHeightChanged { attributionHeight = $0 }
+                .onSingleTapGesture { tapLocation, tapMapPoint in
+                    // Store state and clear selection on tap.
+                    self.tapLocation = tapLocation
+                    self.tapMapPoint = tapMapPoint
+                    clearSelection()
+                }
+                .callout(placement: $calloutPlacement) { placement in
+                    if let feature = placement.geoElement as? Feature {
+                        featureCalloutContent(feature: feature, table: data.featureTable)
+                    } else if let tapMapPoint {
+                        addNewFeatureCalloutContent(table: data.featureTable, point: tapMapPoint)
+                    }
+                }
+                .onNavigatingChanged { _ in
+                    // Reset status when user moves the map.
+                    status = ""
+                }
+                .onViewpointChanged(kind: .centerAndScale) { viewpoint in
+                    // Track current viewpoint.
+                    currentViewpoint = viewpoint
+                }
+                .overlay(alignment: .top) {
+                    instructionsOverlay
+                }
+                .overlay(alignment: .bottom) {
+                    statusOverlay
+                        .padding(.bottom, attributionHeight)
+                }
+                .task(id: tapLocation) {
+                    // Identify when we get a tap location.
+                    guard let tapLocation, let tapMapPoint else { return }
+                    if let identifyResult = try? await mapView.identify(on: data.featureLayer, screenPoint: tapLocation, tolerance: 12),
+                       let geoElement = identifyResult.geoElements.first,
+                       let feature = geoElement as? Feature {
+                        // Place a callout for a feature.
+                        calloutPlacement = .geoElement(geoElement)
+                        data.featureLayer.selectFeature(feature)
+                    } else {
+                        // Place a callout for adding a new feature.
+                        calloutPlacement = .location(tapMapPoint)
+                    }
+                }
+        }
+    }
+    
+    /// A callout that allows the user to add a new feature.
+    @ViewBuilder
+    func addNewFeatureCalloutContent(table: ServiceFeatureTable, point: Point) -> some View {
+        HStack {
+            Text("Create Feature")
+            Button {
+                clearSelection()
+                Task {
+                    try await createFeature(point: point, table: table)
+                }
+            } label: {
+                Image(systemName: "plus.circle")
+            }
+        }
```

Comment:
```suggestion
        Button("Create Feature", systemImage: "plus.circle") {
            clearSelection()
            Task {
                try await createFeature(point: point, table: table)
            }
        }
```

Using the button's initializer here makes it more accessible and uses the default styling.